hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupDelete.java

```diff
@@ -30,6 +31,7 @@
 import org.apache.hadoop.hbase.testclassification.LargeTests;
 import org.apache.hadoop.hbase.util.EnvironmentEdge;
 import org.apache.hadoop.hbase.util.EnvironmentEdgeManager;
+import org.apache.hadoop.thirdparty.com.google.common.collect.Sets;
```

Comment:
This should be using `org.apache.hbase.thirdparty.com.google.common.collect.Sets`, not the hadoop one. Backports to branch-2 fail because hadoop didn't have shaded jars in Hadoop 2.x

Reply:
https://github.com/apache/hbase/pull/5917

Reply:
https://github.com/apache/hbase/pull/5918